Build.mak

```diff
@@ -1,58 +1,33 @@
-# Set of flags required to compile vibe.d without dub
-override DFLAGS += -version=VibeLibeventDriver -version=Have_openssl \
-	-I./submodules/vibed/source -I./submodules/libevent -I./submodules/openssl
+# Binaries and packaging
 
-# Set of flags required to compile D-YAML without dub
-override DFLAGS += -I./submodules/d-yaml/source -I./submodules/tinyendian/source
+$O/pkg-neptune.stamp: $B/neptune-release $B/neptune-overview
 
-# Set of flags required to compile d-colorize without dub
-override DFLAGS += -I./submodules/d-colorize/source
+$B/neptune-overview:
+	dub build :overview
 
-# Compile all vibe.d as a single static library because rdmd can't track
-# all imported dependencies correctly in this case. Static library approach
-# instead of compiling all sources in one go was chosen to reduce recompilation
-# time for actual apps in this repo.
-$O/libvibed.a: $(shell find $C/submodules/vibed/source -type f -name '*.d')
-	$(call exec, dmd $(DFLAGS) -lib -of$@ $?)
+$B/neptune-autopr:
+	dub build :autopr
 
-# Same for D-YAML
-$O/libdyaml.a: $(shell find $C/submodules/d-yaml/source -type f -name '*.d') \
-		$(shell find $C/submodules/tinyendian/source -type f -name '*.d')
-	$(call exec, dmd $(DFLAGS) -lib -of$@ $?)
+$B/neptune-release:
+	dub build :release
 
-$O/libd-colorize.a: $(shell find $C/submodules/d-colorize/source -type f -name '*.d')
-	$(call exec, dmd $(DFLAGS) -lib -of$@ $?)
+all += $B/neptune-release $B/neptune-autopr $B/neptune-overview
 
-################################################################################
+# Wrapper for dub test to be used from CI
 
-$O/allunittests $O/fastunittest: $O/libvibed.a $O/libdyaml.a $O/libd-colorize.a
+dub-unittest:
+	dub test :overview
+	dub test :release
+	dub test :autopr
```

Comment:
I tried that, but it looks like makd tries to do some automatic machinery despite that which fails (allunittests.d generation). Maybe not looked hard enough.

Reply:
I don't know exactly what you tried, but in theory this should work because there is no `$O/unittest-%.stamp` defined in Makd. Maybe it could have failed for `$O/test-%.stamp` because that's already defined so they might compete. If you change that to `$O/inttest-%.stamp` for now I think it should work too.